src/webui/www/private/upload.html

```diff
@@ -14,7 +14,7 @@
     <iframe id="upload_frame" name="upload_frame" class="invisible" title="" src="about:blank"></iframe>
     <form action="api/v2/torrents/add" enctype="multipart/form-data" method="post" id="uploadForm" style="text-align: center;" target="upload_frame" autocorrect="off" autocapitalize="none">
         <div style="margin-top: 25px; display: inline-block; border: 1px solid lightgrey; border-radius: 4px;">
-            <input type="file" id="fileselect" accept=".torrent, application/x-bittorrent" name="fileselect[]" multiple />
+            <input type="file" id="fileselect" name="fileselect[]" multiple />
```

Comment:
@Chocobo1 
Is there a way to do it only for clients that don't properly support it?

Reply:
I think it is possible to do it in js. First detect if client is ios then modify `accept` attribute.

Reply:
Done